unitTests/misc/BackOffTest.ts

```diff
@@ -0,0 +1,31 @@
+import { backOff } from '../../src/misc/BackOff';
+
+export function BackOffTest() {
```

Comment:
Hum seems like we are not testing an awful lot in that utility module.

Can we check that we have sane default ? (eg: 10 number of retry, don't have to pass in a retryConditionOption, etc.).

We should also be checking that all the possible options on that module works correctly.

Eg: number of attempts, delay, retryCondition being called, etc.